com.visualstudio.code.oss.json

```diff
@@ -13940,8 +13940,20 @@
           "type": "file",
           "dest-filename": "lazy_object_proxy-1.3.1-cp35-cp35m-manylinux1_x86_64.whl",
           "url": "https://files.pythonhosted.org/packages/10/88/d7b8274edf912be71d73a1e5de7988f62b63d9e88e591da2e6f921ab69e6/lazy_object_proxy-1.3.1-cp35-cp35m-manylinux1_x86_64.whl",
+          "only-arches": [
+            "x86_64"
+          ],
           "sha256": "7bd527f36a605c914efca5d3d014170b2cb184723e423d26b1fb2fd9108e264d"
         },
+        {
+          "type": "file",
+          "dest-filename": "lazy_object_proxy-1.3.1-cp35-cp35m-manylinux1_i686.whl",
```

Comment:
Can you try avoiding binary whl files and build from source instead?

Reply:
I think this `python_packages` part was created with some generator, because there are a lot of whl packages. I've only added those where the build was failing.

Reply:
Indeed it was: https://github.com/flathub/com.visualstudio.code.oss/blob/master/build.py#L119

Here is another tool that does it without binaries: https://github.com/flatpak/flatpak-builder-tools/tree/master/pip

So it would be nice if somebody did the same here.

Reply:
Ok, done.